src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments?: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments && comments.map((comment) => (
```

Comment:
Is there a chance that `comments` will be just an empty array? 

To double-cover here, let's change this condition to a `comments?.length && ...`. This way you will actually check that there are comments and they are not just an empty array. 
Btw, if comments are optional prop, it's recommended to set a default value, so if there are comments passed as a prop they will always evaluate to an empty array.  And then you have that `comments?.length` condition that will do the job.